compiler_opt/es/blackbox_optimizers.py

```diff
@@ -349,12 +356,12 @@ def normalize_function_values(function_values, current_value):
   return (np.array(normalized_values[:-1]), normalized_values[-1])
 
 
-def mc_gradient(precision_parameter,
-                est_type,
-                perturbations,
-                function_values,
-                current_value,
-                energy=0):
+def mc_gradient(precision_parameter: float,
```

Comment:
might be worth renaming monte_carlo_gradient

Reply:
It has been renamed to monte_carlo_gradient